modules/services/shikane.nix

```
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.shikane;
+  tomlFormat = pkgs.formats.toml { };
+in {
+  meta.maintainers = [ hm.maintainers.therealr5 ];
+  options.services.shikane = {
+    enable = mkEnableOption
+      "shikane, A dynamic output configuration tool that automatically detects and configures connected outputs based on a set of profiles.";
+    package = mkOption {
```

Please add an empty line between each option to help readability.

```suggestion

    package = mkOption {
```